docs/sect-finality.md

```diff
@@ -485,3 +485,268 @@ A Catch-up response message contains critical information for the requester node
+- _BeefyNextAuthoritySetInfo_: It is a tuple consisting of:
+    - ValidatorSetID
+    - Len (u32): length of the validator set
+    - Merkle Root of the list of Next Beefy Authority Set (ECDSA public keys). The exact format depends on the implementation.  
+- Parent Block number and Parent Block Hash.
+- Extra Leaf Data: Currently the Merkle root of the list of (ParaID, ParaHeads)
+:::
+
+
+###### Definition -def-num- Commitment {#defn-beefy-Commitment}
+:::definition
+
+**Signed Commitment:**
+`commitment` is a tuple of `(payload, Block Number, ValidatorSetID)`. A `signed commitment` is a tuple `(commitment, signatures)`, where `signatures` is a list of optional signatures of the validator set on the SCALE encoded `commitment`. Note that the number of signatures in `signatures` may be less than the length of the Validator Set. 
+
+:::
+
+###### Definition -def-num- Witness Data {#defn-beefy-witness-data}
+:::definition
+
+**Signed Commitment Witnesses** contains the commitment and an array indicating which validator of the Polkadot network voted for the statement (but not the signatures themselves). The indicators of which validator voted for the statement are just claims and provide no proof. It also contains the signature of one validator on the commitment, which is used only by the subsampling-based Light Clients. The network message is defined in [Definition -def-num-ref-](chap-networking#defn-grandpa-beefy-signed-commitment-witness) and the relayer saves it on the chain of the remote network.
+
+:::
+###### Definition -def-num- Light Client {#defn-beefy-light-client}
+:::definition
+
+A **light client** is an abstract entity in a remote network such as Ethereum. It can be a node or a smart contract with the intent of requesting finality proofs from the Polkadot network. A light client reads the witness data ([Definition -def-num-ref-](sect-finality#defn-beefy-witness-data) from the chain, then requests the signatures directly from the relayer in order to verify those.
+
+:::
+###### Definition -def-num- Relayer {#defn-beefy-relayer}
+:::definition
+
+A **relayer** (or "prover") is an abstract entity that takes finality proofs from the Polkadot network and makes those available to the light clients. The relayer attempts to convince the light clients that the finality proofs have been voted for by the Polkadot relay chain validators. The relayer operates off-chain and can for example be a node or a collection of nodes.
+
+:::
+### -sec-num- Voting on Statements {#id-voting-on-statements}
+
+The Polkadot Host signs a statement ([Definition -def-num-ref-](sect-finality#defn-beefy-statement)) and gossips it as part of a vote ([Definition -def-num-ref-](chap-networking#defn-msg-beefy-gossip)) to its peers on every new finalized block. The Polkadot Host uses ECDSA for signing the statement since Ethereum has better compatibility for it compared to SR25519 or ED25519.
```

Comment:
The statement definition seems to be missing. Also, the Beefy gossip message does not seem to be defined.

Reply:
i have added the missing links and definitions